### PR TITLE
Add Japanese translation

### PR DESCRIPTION
**Pull request by ariarijp**
Hi, I've added Japanese translation for WBO.

Thank you.

### CODE REVIEW

File: server/translations.json [hunk 1]
```diff
@@ -36,6 +36,26 @@
...
+        "configuration": "設定",
+        "collaborative_whiteboard": "協同作業できるオンラインホワイトボード",
+        "size": "サイズ",
+        "zoom": "拡大・縮小",
+        "tools": "ツール",
+        "rectangle": "矩形",
+        "menu": "メニュー",
+        "text": "テキスト",
+        "straight_line": "直線",
+        "pencil": "ペン",
```
Comment (ariarijp) [inline server/translations.json:52]: "ペン" means "Pen", but I think "ペン" is more understandable for Japanese users than "鉛筆(Pencil)"

**Comment by lovasoa** [comment]
Awesome, thank you for your contribution ! Could you also translate the other strings we have for the index page ?

```json
{
        "hand": "Hand",
        "loading": "Loading",
        "tagline": "A free and opensource online collaborative drawing tool. Sketch new ideas together on WBO !",
        "configuration": "Configuration",
        "collaborative_whiteboard": "Collaborative whiteboard",
        "size": "Size",
        "zoom": "Zoom",
        "tools": "Tools",
        "rectangle": "Rectangle",
        "menu": "Menu",
        "text": "Text",
        "straight_line": "Straight line",
        "pencil": "Pencil",
        "click_to_zoom": "Click to zoom in\nPress shift and click to zoom out",
        "keyboard_shortcut": "keyboard shortcut",
        "opacity": "Opacity",
        "color": "Color",
        "eraser": "Eraser",
        "index_title": "Welcome to WBO !",
        "introduction_paragraph": "WBO is a <a href=\"https://github.com/lovasoa/whitebophir\" title=\"Free as in free speech, not free beer. This software is released under the GPL license\">free and open-source</a> online collaborative whiteboard that allows many users to draw simultaneously on a large virtual board. The board is updated in real time for all connected users, and its state is always persisted. It can be used for many different purposes, including art, entertainment, design and teaching.",
        "share_instructions": "To collaborate on a drawing in real time with someone, just send them its URL.",
        "public_board_description": " The <b>public board</b> is accessible to everyone. It is a happily disorganized mess where you can meet with anonymous strangers and draw together. Everything there is ephemeral.",
        "open_public_board": "Go to the public board",
        "private_board_description": "You can create a <b>private board</b> with a random name, that will be accessible only by its link. Use this if you want to share private information.",
        "create_private_board": "Create a private board",
        "named_private_board_description": "You can also create a <strong>named private board</strong>, with a custom URL, that will be accessible to all those who know its name.",
        "board_name_placeholder": "Name of the board...",
        "view_source": "Source code on github"
    }
```

File: server/translations.json (hunk 1 above)
Comment (lovasoa) [reply server/translations.json:52]: OK. The goal is to make the translation feel right, so not being a literal translation from english is perfectly OK.

**Comment by ariarijp** [comment]
@lovasoa Thank you for your review, of course, I did it!
Could you check it again?

**Comment by ariarijp** [comment]
Here is a screenshot of these changes.

スクリーンショット 2020-04-28 1 27 00

**Comment by lovasoa** [comment]
Thank you for your contribution @ariarijp ! 🙏

**Comment by lovasoa** [comment]
I just pushed the changes to production.

https://wbo.ophir.dev/?lang=ja

🚀

**Comment by ariarijp** [comment]
Awesome!

**Pull request merged**